portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,22 +36,65 @@
 @WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
-  private ArrayList<String> commentSection = new ArrayList<String>();
+//   private ArrayList<String> commentSection = new ArrayList<String>();
+
+//   @Override
+//   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+//     response.setContentType("application/json;");
+//     String json = new Gson().toJson(commentSection);
+//     response.getWriter().println(json);
+//   }
+
+//   @Override
+//   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
+//     String comment = request.getParameter("comment");
+//     commentSection.add(comment);
+//     response.sendRedirect("/blog.html");
+//   }
+//   private String getComment(HttpServletRequest request) {
+//     return request.getParameter("comment");
+//   }
```

Comment:
Let's submit only ready code. Either remove commented code or de-comment it please.

Reply:
sorry, thought i included it in the push. done